apps/evm/src/clients/api/queries/getPendingRewards/index.ts

```diff
@@ -114,18 +114,15 @@ const getPendingRewards = async ({
     ...isolatedPoolRewardTokenAddresses,
     ...primeRewardTokenAddresses,
   ]);
-  const rewardTokenPricesPromises = Promise.allSettled(
-    rewardTokenAddresses.map(rewardTokenAddress =>
-      resilientOracleContract.getPrice(rewardTokenAddress),
-    ),
-  );
-
-  const rewardTokenPricesResults = await rewardTokenPricesPromises;
+  const tokenPriceMantissaMapping = await getApiTokenPrice(rewardTokenAddresses);
```

Comment:
Rather than passing `getApiTokenPrice` through props, it should be imported (so that we follow the codebase's practices). You can then remove the `useGetApiTokenPrice` hook, and mock the function in tests.